### PR TITLE
Adding How to commit to Contributor Quickstart

### PR DESCRIPTION
**Pull request by Srabasti**
---
**^ Add meaningful description above**
Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/main/contributing-docs/05_pull_requests.rst#pull-request-guidelines)** for more information.
In case of fundamental code changes, an Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvement+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in a newsfragment file, named `{pr_number}.significant.rst` or `{issue_number}.significant.rst`, in [newsfragments](https://github.com/apache/airflow/tree/main/newsfragments).

### CODE REVIEW

**Review by jedcunningham — APPROVED** [review]

**Review by Srabasti — COMMENTED** [review]
Removed trailing spaces in Quickstart documentation

**Review by jedcunningham — APPROVED** [review]

**Comment by ephraimbuddy** [comment]
Conflicts...

File: contributing-docs/10_working_with_git.rst [hunk 1]
```diff
@@ -194,11 +208,14 @@ Useful when you understand the flow but don't remember the steps and want a quic
...
 
     git fetch --all
+    git add .
+    git commit
     git merge-base my-branch apache/main
     git checkout my-branch
     git rebase HASH --onto apache/main
     git push --force-with-lease
 
+
```
Comment (Srabasti) [inline contributing-docs/10_working_with_git.rst]:
```suggestion
```

**Review by Srabasti — COMMENTED** [review]
Doc update for Quickstart

**Review by vincbeck — APPROVED** [review]

**Pull request merged**